.gitignore

```diff
@@ -21,9 +21,21 @@ moc_*.cpp
 qrc_*.cpp
 ui_*.h
 Makefile*
-*build-*
+*build*
 /Photo
 
 # QtCreator
 
 *.autosave
+
+# VSCode
+
+*.vscode
+
+# Mac
+
+.DS_Store
+
+# Personal need
+
+backlog
```

Comment:
no newline at the end of file
here and elsewhere

Reply:
Added with
https://github.com/sersajur/QPentago/pull/11/commits/15b4eb91f09dbc6bdf4bfd9ac84c80d8c2a4be72